### DISCUSSION TITLE
프로젝트: KAITA 웹사이트

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#9] [3. 프로젝트 아이디어]
설명: 미국이 되서 힘든게 KAITA 이름으로 되어 있는 웹사이트 하나. 안드로이드, AI, Mobile 등등 로 확장 가능. 

테크 스택: 클라우드, HTML, CSS, Javascript, Front 는 모바일/웹, 데이터베이스

부가 설명: 한인 최대 정보 웹사이트!!
개인의 의견을 공유하는 곳은 많으나(관리 어려움), 잘 정리된 웹사이트의 부재...
 * AI로 웹사이트에서 데이터를 수집해서 공유
 * mobile version 제공
 * 광고 수익 창출등.... 
 

https://github.com/Korean-American-IT-Association-KAITA

**Comment by ellijs** [comment]
웹사이트를 만들면 저희 모임이 활성화 되고 발전하는데 도움이 될 것 같습니다. 좋은 아이디어인 것 같습니다. :)

**Comment by klee5760** [comment]
서로 어려움이 있을때 같이도와서 공부를 할수 있다면 추천합니다!
특히 코딩 경력이 많지 안은 분들에게는 엄청난 도움이 됩니다.
